src/Microsoft.Graph.Core/Properties/AssemblyInfo.cs

```diff
@@ -19,15 +18,15 @@
 // Version information for an assembly consists of the following four values:
 //
 //      Major Version
-//      Minor Version 
+//      Minor Version
 //      Build Number
 //      Revision
 //
-// You can specify all the values or you can default the Build and Revision Numbers 
+// You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
```

Comment:
Hmm... because I thought there was a 1.1 already and someone forgot to bump the version. Turns out I saw a 1.0.1 and 1.0.2.

Fixed.